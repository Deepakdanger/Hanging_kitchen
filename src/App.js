
function App() {
  return (
    <div className="App">
      Hello Welcome
    </div>
  );
}

export default App;