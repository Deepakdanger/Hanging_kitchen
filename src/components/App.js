import React from 'react';
import Categorylist from '../containers/Categorylist';
import Lists from '../containers/Lists';
import FoodContainer from '../containers/FoodContainer';

const App = () => (
  <div className="app">
    <Lists />
    <Categorylist />
    <FoodContainer />
  </div>
);

export default App;
