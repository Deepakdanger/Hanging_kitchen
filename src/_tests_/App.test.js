import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import App from '../components/App';
import store from '../reducers/index';

describe('Testing App with redux', () => {
  it('App snapshot testing', () => {
    const app = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>,
    ).toJSON();
    expect(app).toMatchSnapshot();
  });
});
