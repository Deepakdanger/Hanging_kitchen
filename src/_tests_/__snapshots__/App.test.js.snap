// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App with redux App.js snapshot testing 1`] = `
<div
  className="app"
>
  <div
    className="listss"
  >
    <p
      className="select1"
    >
      Select Category
    </p>
    <p
      className="select3"
    >
      SELECT CATEGORY 
    </p>
    <p
      className="select2"
    >
      FROM
    </p>
    <p
      className="select2"
    >
      HERE
    </p>
    <p
      className="select2"
    >
      = = = &gt; 
    </p>
    <div
      className="lists"
    >
      <p>
        click
      </p>
    </div>
  </div>
  <div
    className="meals"
  >
    <p
      className="meals_title"
    >
      List of Meals :
      
    </p>
    <div
      className="meals_discription"
    >
      <p>
        hello111
      </p>
    </div>
  </div>
</div>
`;
